test/functionalTest/cases/2015_notification_templates/notification_templates_cache_refresh.test

```diff
@@ -185,6 +185,7 @@ Date: REGEX(.*)
                 "url": "http://127.0.0.1:9997/${NOTIFY_URL}"
             },
             "lastNotification": "REGEX(.*)",
+            "lastSuccess": "REGEX(.*)",
```

Comment:
As discussed on Skype. The fix is for both cases, not only for 'cache enabled'

NTC